source/_integrations/frontend.markdown

```diff
@@ -116,6 +116,42 @@ For more details about splitting up the configuration into multiple files, see [
 
 Check our [community forums](https://community.home-assistant.io/c/projects/themes) to find themes to use.
 
+## Theme variables
+
+<div class='note warning'>
+This list is incomplete and should only serve as a starting point. We try to extend this list and keep it up-to-date, but the definitive list of supported variables is the [frontend code](https://github.com/home-assistant/frontend) itself.
```

Comment:
Should this even be here?
Is there a plan on what variables should be supported?
I thought the plan was to start using the gallery for this?
Have there been discussions surrounding theme variables that I have not seen? if so can you point me to it?

Reply:
Yes, I think the gallery would be good place for this. Also, we will probably adopt the variables from Material You as supported variables.

Reply:
1. Do we have a public running gallery somewhere, because it would not help if everyone that wants to create a theme needs a local dev instance to spin up a local gallery instance?
And even if we have that: I thought the gallery is for showcasing individual UI components and not writing documentation?

2. Material You should not impact our custom HA CSS vars, right?

3. @ludeeus What do you mean with "supported variables"? The ones we have in the coding are the ones we support. Or am I missing your point?

Reply:
1) <https://home-assistant-gallery.netlify.app/#>, it should be both (showcase variants of the element, and documentation on how to interface with it)
2) :shrug: 
3) "Supported variables" are variables we indicate (by documenting it) that they are stable and if we change/remove them we have to make that a breaking change.

Reply:
1. @bramkragten @ludeeus How should/could that then look like in the gallery? Do you envision a new section with dedicated "demos" that contain a single UI element and then below list all the variables and an explanation of what they affect? We could even add a toggle in the header (similar to what we already have for the dark / light switch) to toggle between the default theme look and an extreme one (rater ugly / colorful) that really clearly show the effect of the theme vars, e.g. different colors for the toggle switch track and circle.

2. @bramkragten Do you know if and how that will impact us from a theming / CSS perspective? Not clear to me currently.

3. Reg. "supported variables". In my mind, part of the effort to document them is to verify them by writing them down and fixing any gaps or ambiguities to reach such a stable set.

Reply:
@matthiasdebaat have some ideas about 1